templates/bare/package.json

```diff
@@ -35,6 +35,7 @@
     "@aragon/apps-vault": "^4.1.0",
     "@aragon/apps-voting": "^2.1.0",
     "@aragon/apps-finance": "^3.0.0",
+    "@aragon/apps-survey": "^1.0.0",
```

Comment:
I don't think these other templates need the survey dependency?

Reply:
I used to have problems with this as well, for some reason the templates using the `BaseTemplate` were asking for all the apps being used by the base :/
Will merge and polish it if possible

Reply:
I was experiencing the problem of the templates not compiling if the Survey app was not added. Same thing happened when I was adding the Payroll app.